### ISSUE TITLE
The "Nothing here" icon with the sad face is too sad and looks like an error!

### ISSUE THREAD

**Issue by GaryWilletts** [msupply-foundation/open-msupply#2687]
## Is your feature request related to a problem? Please describe 👀
<img width="156" alt="Screenshot 2024-01-09 at 07 09 52" src="https://github.com/msupply-foundation/open-msupply/assets/11517184/0443e85c-8fad-4913-96bb-44a399691695">

The "Nothing here" icon with the sad face is too sad and looks like an error! It always make me feel like I did something wrong. And I think it makes other users or customers feel the same.

## Describe the solution you'd like 🎁
A different image that says something like "Nothing here yet". The image should make it clear that his isn't an error or a bad condition but just a natural part of the process. 

When doing this issue please standardise all of the modals as suggested here:
https://github.com/msupply-foundation/open-msupply/issues/6724

**Comment by adamdewey** [comment]
I do agree with Gary to be honest!

**Comment by DhanyaHerath** [comment]
The Designer said it is Good, and it is subjective so we will have to leave it as is! Thanks for the input though :)

**Comment by adamdewey** [comment]
Every time I open a page I get a little bit sadder :(

**Comment by adamdewey** [comment]
What about something a bit more neutral, like a tumbleweed or a nice serene nightime desert scene

![image](https://github.com/msupply-foundation/open-msupply/assets/19318937/0a5a6891-01d6-419e-9974-5ff3b0644309)

Imagine that - you could feel nice and relaxed when using omSupply!

Instead of... well, a bit sad

**Comment by GaryWilletts** [comment]
@DhanyaHerath I object! Seriously, I think the designer is wrong and doesn't realise the impact it has on users. Even just changing the face to a smile would be better. But a negative, sad face is all wrong. Just makes the user feel like there's something wrong, that they did something wrong or that something is broken. Bad look, bad vibe. If the designer insists then I'd like them to justify why it's good.

**Comment by mark-prins** [comment]
@GaryWilletts I had raised with Mariya - and when we get options will run past you & Adam 😉

**Comment by mariyamsupply** [comment]
Hey! How about something subtle and neutral? Seedling represents the opportunity for growth and creation.
This is visually nice and should be pretty simple to just swap the image.

<img width="1031" alt="No items" src="https://github.com/user-attachments/assets/6b414dba-a27b-4385-81f6-bc199b016957" />

**Comment by mark-prins** [comment]
Here's an svg - let's get this issue done 😄 The current image is an imported gif, this can now be a standard svgicon

![Image](https://github.com/user-attachments/assets/04efdafa-e0d0-46e4-a782-f97b870a8da5)

**Comment by DhanyaHerath** [comment]
When doing this issue please standardise all of the modals as suggested here: 
https://github.com/msupply-foundation/open-msupply/issues/2687

**Comment by mariyamsupply** [comment]
This one? https://github.com/msupply-foundation/open-msupply/issues/6724

**Comment by DhanyaHerath** [comment]
Yes thanks @mariyamsupply!  will update the comment and the issue
